core/tests/protractor.conf.js

```diff
@@ -61,102 +63,102 @@ exports.config = {
     // them. Therefore, we mention the complete directory
     // in 'full'.
     full: [
-      'protractor/*.js',
-      'protractor_desktop/*.js'
+      'local_compiled_js/core/tests/../../protractor/*.js',
```

Comment:
Ditto: does this work when you run with --prod_env?

Reply:
I have added a compile option in build which compiles ts files to `local_compiled_js` whenever frontend or e2e test are run. This ensures that local_compiled_js is present irrespective of prod_mode when we are testing. So, yes it works in prod mode also.